src/pyop/storage.py

```diff
@@ -120,10 +136,21 @@ class RedisWrapper(StorageBase):
     Supports JSON-serializable data types.
     """
 
-    def __init__(self, db_uri, collection, ttl=None):
+    def __init__(
+        self, db_uri, *, db_name=None, collection, ttl=None, extra_options=None
```

Comment:
Note: set all but the first arg to RedisWrapper to be keyword-only args.

Added `db_name` as the redis db index.

Reply:
Now, the MongoWrapper and RedisWrapper initialization interfaces are the same.